errors/errors.go

```diff
@@ -53,4 +53,5 @@ var (
 	ErrRegistryNoContent       = errors.New("sync: could not find a Content that matches localRepo")
 	ErrSyncSignatureNotFound   = errors.New("sync: couldn't find any upstream notary/cosign signatures")
 	ErrSyncSignature           = errors.New("sync: couldn't get upstream notary/cosign signatures")
+	ErrUnexpectedError         = errors.New("error: unexpected error")
```

Comment:
Where is this actually used?

Reply:
I use it just in tests to be more expressive on the meaning of the error I test for.

Reply:
Then move this error into the tests itself. It doesn't belong here.

Reply:
maybe create a new errors/test_errors.go and move it there.

Reply:
Ok! I have resolved this issue.